Makefile

```diff
@@ -110,7 +110,7 @@ test: manifests generate fmt vet envtest godeps-update ## Run tests.
 
 
 OPERATOR_NAMESPACE ?= openshift-storage
-TOPOLVM_CSI_IMAGE ?= quay.io/ocs-dev/topolvm:latest
+TOPOLVM_CSI_IMAGE ?= quay.io/topolvm/topolvm:0.15.0
```

Comment:
This image does not exist upstream which is why the CI is failing.
Lets stick to quay.io/ocs-dev/topolvm:latest